src/main/java/com/thebois/controllers/info/InventoryController.java

```diff
@@ -19,15 +23,20 @@
     /**
      * Instantiates the controller with the view and a colony reference.
      *
-     * @param inventoryView The view to be updated.
-     * @param inventory     The inventory to get information from, e.g. item counts.
+     * @param inventory The inventory to get information from, e.g. item counts.
+     * @param uiSkin    The skin to style widgets with.
      */
-    public InventoryController(final InventoryView inventoryView, final IInventory inventory) {
-        this.inventoryView = inventoryView;
+    public InventoryController(final IInventory inventory, final Skin uiSkin) {
+        this.inventoryView = new InventoryView(uiSkin);
         this.inventory = inventory;
         itemCounts = new HashMap<ItemType, Integer>();
     }
 
+    @Override
+    public IActorView getView() {
+        return inventoryView;
+    }
+
```

Comment:
Can't mark the code you haven't changed, but the `update` method below:

Missing `@Override` and then you can remove the javadoc.